Content.Server/Projectiles/ProjectileSystem.cs

```diff
@@ -45,6 +49,12 @@ private void OnStartCollide(EntityUid uid, ProjectileComponent component, ref St
         RaiseLocalEvent(uid, ref ev);
 
         var otherName = ToPrettyString(target);
+        var damageRequired = _destructibleSystem.DestroyedAt(target);
+        if (TryComp(target, out DamageableComponent? damageableComponent))
+        {
+            damageRequired -= damageableComponent.TotalDamage;
+            damageRequired = FixedPoint2.Max(damageRequired, FixedPoint2.Zero);
+        }
         var modifiedDamage = _damageableSystem.TryChangeDamage(target, ev.Damage, component.IgnoreResistances, origin: component.Shooter);
```

Comment:
```suggestion
        var modifiedDamage = _damageableSystem.TryChangeDamage(target, ev.Damage, component.IgnoreResistances, damageable: damageableComponent, origin: component.Shooter);
```
we can reuse this here so the resolve won't have to get this